include/hw/ssi/esai.h

```diff
@@ -0,0 +1,71 @@
+/*
+ * BSD 3 Clause - See LICENCE file for details.
+ *
+ * Copyright (c) 2015, Intel Corporation
+ * All rights reserved.
+ */
```

Comment:
Not sure about this. If you copied this from an Intel file we can still keep Intel license, but we also need to add

```Copyright 2020 NXP```

ESAI driver looks good to me. I have some concerns about SAI